client/src/__snapshots__/Dates.spec.js.snap

```diff
@@ -0,0 +1,87 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Dates Component should render 1`] = `
+<div
+  className="expectedDates"
+>
+  <div
+    className="free"
+  >
+    <span
+      className="bold"
+    >
+      Get it
```

Comment:
Why is this spread out on like 30 lines?

Reply:
Not exactly sure, should I edit the format?